packs/feats/shining-oath.json

```
@@ -31,7 +28,45 @@
             "remaster": false,
             "title": "Pathfinder Core Rulebook"
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "damage",
+                "key": "RollOption",
+                "label": "Retributive Strike against an undead",
+                "option": "shining-oath",
+                "predicate": [
+                    "feature:paladin"
+                ],
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "label": "Shining Oath",
+                "predicate": [
+                    "shining-oath",
+                    "target:mode:undead"
+                ],
```

```suggestion
                "predicate": [
                    "retributive-strike",
                    "feature:paladin",
                    "target:mode:undead"
                ],
```
I'll push up a PR to add the retributing strike toggle for paladins